.travis.yml

```diff
@@ -2,7 +2,8 @@ notifications:
   email: false
 language: go
 go:
-- 1.9
+- 1.8.x
```

Comment:
As we use Build Stages it's more tricky. You can check [this issue](https://github.com/travis-ci/beta-features/issues/28) for possible solutions. We would like to run `Lint` and `Test unit and integration` on both. The rest of stages can execute on `1.9.x` only.